moped-database/hasura-cluster

```diff
@@ -707,6 +291,57 @@ function show_banner() {
   echo "- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -";
 }
 
+function __check_export_complete() {
```

Comment:
This replication system will check the end of the export files it keeps around to make sure that they didn't get interrupted while being downloaded.

Reply:
nice touch to close the loop on this issue we see with the VZ stack every once in a while. 🙏